package.json

```diff
@@ -182,5 +182,8 @@
   "devEngines": {
     "node": "6.x",
     "npm": "3.x"
+  },
+  "dependencies": {
+    "eslint-plugin-fp": "^2.2.0"
```

Comment:
Did not notice this, i needed to install some things because the linter did not work.